### ISSUE TITLE
The manuel added extensions are getting removed after the every app update

### ISSUE THREAD

**Issue by kyoyacchi** [kiwibrowser/src.next#410]
Like as i said, i _was_ added Universal Bypass and Youtube Background ([from here](https://github.com/kiwibrowser/android/issues/60)) extensions to kiwi browser.

When you update app they _are_ being gone, there is no problem about the other extensions (installed from chrome store)

I forgot to record 🤦, but you can see the same result if you do as i did.

**Comment by gnadelwartz** [comment]
did you install as zip or as crx?

**Comment by blogmangit** [comment]
> that installing from zip is removed seems to be a bug in kiwi and installing as crx is a workaround.
> 
> @kiwibrowser it was also mentioned (and confirmed) on discord that extensions installed from zip are vanishing on clearing the browser cache while extensions installed from crx will stay. any idea why?

https://github.com/SomMeri/grunt-zip-to-crx

**Comment by blogmangit** [comment]
> @gnadelwartz then what's the point of installing from zip? also i can not install universal bypass as crx

https://github.com/SomMeri/grunt-zip-to-crx

**Comment by tachibana-shin** [comment]
I don't think this is because Kiwi it affects all Chrome based browsers including Chrome itself on Windows. A note using `.crx` is probably a good thing

**Comment by tachibana-shin** [comment]
if someone wants now i can create a website that converts online `zip` to `crx` or `xpi` format
